crates/ruff_python_ast/ast.toml

```diff
@@ -446,44 +465,44 @@ doc = "See also [Attribute](https://docs.python.org/3/library/ast.html#ast.Attri
 fields = [
   { name = "value", type = "Expr" },
   { name = "attr", type = "Identifier" },
-  { name = "ctx", type = "ExprContext" },
+  { name = "ctx", type = "ExprContext", skip_source_order = true },
 ]
 
 [Expr.nodes.ExprSubscript]
 doc = "See also [Subscript](https://docs.python.org/3/library/ast.html#ast.Subscript)"
 fields = [
   { name = "value", type = "Expr" },
   { name = "slice", type = "Expr" },
-  { name = "ctx", type = "ExprContext" },
+  { name = "ctx", type = "ExprContext", skip_source_order = true },
 ]
 
 [Expr.nodes.ExprStarred]
 doc = "See also [Starred](https://docs.python.org/3/library/ast.html#ast.Starred)"
 fields = [
   { name = "value", type = "Expr" },
-  { name = "ctx", type = "ExprContext" },
+  { name = "ctx", type = "ExprContext", skip_source_order = true },
 ]
 
 [Expr.nodes.ExprName]
 doc = "See also [Name](https://docs.python.org/3/library/ast.html#ast.Name)"
 fields = [
-  { name = "id", type = "Name" },
-  { name = "ctx", type = "ExprContext" },
+  { name = "id", type = "Name", skip_source_order = true },
+  { name = "ctx", type = "ExprContext", skip_source_order = true },
```

Comment:
I'd be fine to teach the script to skip `ExprContext` and `Name` by default.

Reply:
I agree. After these only `IpyEscapeKind` and `Number` was remaining. I felt it's good to move those types to code to? this way we don't need the skip for now.